hazelcast/src/main/java/com/hazelcast/internal/dynamicconfig/DynamicConfigXmlGenerator.java

```diff
@@ -401,6 +402,17 @@ public static void pnCounterXmlGenerator(ConfigXmlGenerator.XmlGenerator gen, Co
         }
     }
 
+    public static void externalDataStoreXmlGenerator(ConfigXmlGenerator.XmlGenerator gen, Config config) {
```

Comment:
method is not used in this file, is it correct?

Reply:
It's used in `DynamicConfigGeneratorUtil#externalDataStoreConfigGenerator`